test/e2e/main_test.go

```diff
@@ -37,6 +39,14 @@ import (
 // Features within an area may be split across different test functions.
 const LabelArea = "area"
 
+// LabelInstallCrossplane is used to mark tests that are going to install
+// Crossplane.
+const LabelInstallCrossplane = "install-crossplane"
```

Comment:
Would something like "invokes Helm" be clearer here? It's a little surprising that "Installs Crossplane" also includes the cases where we Helm upgrade to turn on a feature flag.

Reply:
I wasn't too sure about the name either. It's actually mostly about modifying the crossplane installation's status (installing/upgrading/uninstalling), so I wouldn't specifically mention helm maybe, as it's just an implementation detail.
Maybe "modify-crossplane-config" or "modify-crossplane-installation"? It would make it also less redundant with the "install-crossplane" flag above.

Reply:
I like `modify-crossplane-installation`.